Mathlib/AlgebraicGeometry/Restrict.lean

```diff
@@ -177,81 +177,104 @@ lemma Scheme.map_basicOpen' (r : Γ(U, ⊤)) :
   exact PresheafedSpace.IsOpenImmersion.ofRestrict_invApp _ _ _
 
 lemma Scheme.Opens.ι_image_basicOpen (r : Γ(U, ⊤)) :
-    U.ι ''ᵁ (U.toScheme.basicOpen r) = X.basicOpen r := by
-  rw [Scheme.map_basicOpen', Scheme.basicOpen_res_eq]
+    U.ι ''ᵁ U.toScheme.basicOpen r = X.basicOpen r := by
+  rw [Scheme.map_basicOpen, Scheme.basicOpen_res_eq]
 
 lemma Scheme.map_basicOpen_map (r : Γ(X, U)) :
     U.ι ''ᵁ (U.toScheme.basicOpen <| U.topIso.inv r) = X.basicOpen r := by
   simp only [Scheme.Opens.toScheme_presheaf_obj]
-  rw [Scheme.map_basicOpen', Scheme.basicOpen_res_eq, Scheme.Opens.topIso_inv,
+  rw [Scheme.map_basicOpen, Scheme.basicOpen_res_eq, Scheme.Opens.topIso_inv,
     Scheme.basicOpen_res_eq X]
 
+/-- If `U ≤ V`, then `U` is also a subscheme of `V`. -/
+protected noncomputable
+def Scheme.homOfLE (X : Scheme.{u}) {U V : X.Opens} (e : U ≤ V) : (U : Scheme.{u}) ⟶ V :=
+  IsOpenImmersion.lift V.ι U.ι (by simpa using e)
```

Comment:
Should we enforce using `(U : Scheme.{u})` over `U.toScheme`? I know there are sometimes issues with `(_ : Set X)` and `_.carrier` where the first seems to be preferred? I am asking because below you sometimes write `U.toScheme`.

Reply:
I believe they are exactly the same after the elaborator.